inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/LoadNodeUtils.java

```diff
@@ -117,6 +118,7 @@ public static LoadNode createLoadNode(StreamSink streamSink, Map<String, StreamF
                 .collect(Collectors.toList());
         List<FieldRelation> fieldRelations = parseSinkFields(streamSink.getSinkFieldList(), constantFieldMap);
         Map<String, String> properties = streamSink.getProperties().entrySet().stream()
+                .filter(v -> Objects.nonNull(v.getValue()))
```

Comment:
Suggest use `StringUtils.isNotBank()`.

Reply:
> Suggest use `StringUtils.isNotBank()`.

Value is of type Object，so i use Objects.nonNull(Object obj)